src/Components/Table/Columns.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+const Columns = props => {
+  return (
+    <thead className="thead-dark">
+      <tr>
+        {Object.keys(props.fakeDataCol).map((propriety, index) => {
+          const colName = propriety;
+          return (
+            <th scope="col" key={index}>
+              {colName.charAt(0).toUpperCase() + colName.slice(1)}
+            </th>
+          );
+        })}
```

Comment:
I like what you did, quite clever.

Prop tip: When creating codes that are a bit more elaborate, try to move it to a function and then use the function inside the component. I will give you an example:

```js

const formatHeaders = (dataColumns) => {
   return Object.keys(dataColumns).map((colName, index) => {
          return (
            <th scope="col" key={index}>
              {colName.charAt(0).toUpperCase() + colName.slice(1)}
            </th>
          );
        });
}

import React from "react";

const Columns = props => {
  return (
    <thead className="thead-dark">
      <tr>
        {formatHeaders(props.fakeDataCol)}

```

Reply:
Thank you! I will change it